app/controllers/sessions_controller.rb

```diff
@@ -28,11 +30,37 @@ def create
     redirect_to url
   end
 
+  def lti_launch
+    auth_hash = request.env["omniauth.auth"]
```

Comment:
Mind calling this `auth_obj` or something other than `auth_hash`? It looks like it's not actually a hash based on the `.uid` below.

Reply:
No problem! Though we also name this same variable ‘auth_hash’ in the action relying upon the standard GitHub OmniAuth strategy (in the ‘sessions#create’ action) — I believe both are based on the standard OmniAuth authentication object, which they (perhaps misleadingly) [title the “Auth Hash”](https://github.com/omniauth/omniauth/wiki/Auth-Hash-Schema). 

Should I change the variable name in ‘sessions#create’ too for consistency?

Reply:
Ah dang. Yeah, if thats the standard, lets do `auth_hash`. Good call :)